circle.yml

```diff
@@ -1,6 +1,7 @@
 dependencies:
   pre:
     - sudo add-apt-repository -y ppa:ethereum/ethereum; sudo apt-get update; sudo apt-get install -y solc
+    - git clone https://github.com/input-output-hk/sbt-verify.git; cd sbt-verify; sbt publishLocal
```

Comment:
This is not a blocker at all but this way we are trusting that no one has compromised this repo and default branch. We need to keep an eye on it and check it's permissions.

> Good security means trying to limit the damage a Trusted role can do, even if you can't completely eliminate the Trusted role. [1](https://www.linkedin.com/pulse/id-known-what-we-were-starting-ray-dillinger/)

Reply:
You're right...I wonder if it would be possible to include a jar with the plugin in our repository itself